common/request.go

```diff
@@ -264,5 +264,8 @@ func (r *Request) Timing() goja.Value {
 
 // URL returns the request URL
 func (r *Request) URL() string {
+	if r == nil {
```

Comment:
Same question as for the `nil` checks in `Frame`.

Reply:
https://github.com/grafana/xk6-browser/pull/174/files#r779340558